notes.md

```diff
@@ -0,0 +1,124 @@
+
+  - Price per share is denominated in USD.
+  - CO2KN, INC will announce its own exchange rate at some point close to the auction.
+  - So, no need for an oracle. This is optimal because the IRS sees each transaction of ETH as a sale. So it's not viewed as one transaction of ETH for Equity, it's seen as USD for equity and then cash for ETH.
+  - Nature of securities takes the following values: `equity`, `debt`, `convertible`, etc.
+  - 'Closing soon' can probably be posted a week before close.
+
+## also need to record
+
+* Name of the shareholder;
+* Complete mailing address of the stock shareholder including contact number;
+* Stock certificate number;
+* The total number of shares outstanding;
+* The date the shares were purchased;
+* How much the person paid per share (consideration/monetary value);
+* The class of shares;
+* Why the shares were transferred.
+
+### implementation
+
+* The `shareholder name`, their `mailing address`, and `contact number` MUST all be stored in an off-chain database and the content hashed, and the `hash` associated with the address.
+* The total number of shares outstanding is the difference between a cap and `token.totalSupply()` and is rightly a function of the `CrowdsaleContract`.
+* The date the share was purchased is a property of the block the purchase was processed in so there is no need to store it in the token. However the user database might wish to record that information as well for convenience.
+* The price the user paid is a function of the ETH price per Token (which can change during the crowdsale) and ETH price at that point (which can also be set in the `CrowdsaleContract`) and the number of ETH spent, which the `CrowdsaleContract` also knows and which is returned in the transaction receipt.
+* With regard to the certificate number, the token should record the certificate numbers, either algorithmically if possible, or else it will need to be generated offchain and imported in bulk when the certificates themselves are issued.
+* A certificate might represent more than one share, in which case it will need to record a range of inputs in the form of an encoded bytes array.
+* as shares are traded a script needs to pull Transfer events from the global blockchain and keep the off-chain records up to date, either in real-time, or in an ad-hoc way.  This should be the job of the Crowdsale Admin website.
+
+### Questions and Scenarios.
+
+1. When someone buys a Token does that mean they have bought a specific Stock with a specific certificate number, or does it mean they've bought the right to be issued a stock certificate?
+
+    scenario: Token Buys Share immediately.
+    scenario: Token Buys rights to a share, when the crowdsale is finished.
+
+2. If I buy 100 Tokens would that result in the issuance of 100 certificates, or one certificate with 100 shares on it.
+
+    scenario: One token = one certificate = one share.
+    scenario: One token = one certificate has many shares.
```

Comment:
Maybe keeping track of the stock certificate computed off-chain?  One token = one share and the certificate is updated on the backend

Reply:
@davesag I think @tavakyan's suggestion would make sense